src/modules/vod_chat/index.js

```diff
@@ -3,13 +3,80 @@ const chat = require('../chat');
 const nicknames = require('../chat_nicknames');
 const watcher = require('../../watcher');
 const colors = require('../../utils/colors');
+const chatSettings = require('../chat_settings');
 
 const CHAT_MESSAGE_SELECTOR = '.video-chat__message span[data-a-target="chat-message-text"]';
 const CHAT_FROM_SELECTOR = '.video-chat__message-author';
 const CHAT_USER_SELECTOR = '.chat-author__display-name,.chat-author__intl-login';
 const SCROLL_INDICATOR_SELECTOR = '.video-chat__sync-button';
 const SCROLL_CONTAINER_SELECTOR = '.video-chat__message-list-wrapper';
 const COLOR_REGEX = /rgb\(([0-9]+), ([0-9]+), ([0-9]+)\)/;
+const CHAT_HEADER = '.video-chat__header';
+const CHAT_SETTINGS_BUTTON_SELECTOR = '.bttv-vod-settings-button';
+const CHAT_SETTINGS_WINDOW_CLOSE_BUTTON_SELECTOR = '.bttv-vod-chat-close-button';
+const CHAT_SETTINGS_WINDOW_SELECTOR = '.bttv-vod-settings-window';
+const CHAT_SETTINGS_BUTTON_TEMPLATE = `
+<div class="tw-absolute tw-mg-r-1 tw-right-0 bttv-vod-settings-button">
```

Comment:
these templates seem to rely heavily on twitch style classes.. is there a way to make something more barebones which won't break as easily if twitch changes their styles? i assume this is already broken due to the many site changes since this was opened, for example.